docs/shared/coproc-typical-config.mdx

```diff
@@ -4,26 +4,46 @@ coprocessor:
   timeout: 2s # The timeout for all coprocessor requests. Defaults to 1 second (1s)
   router: # This coprocessor hooks into the `RouterService`
     request: # By including this key, the `RouterService` sends a coprocessor request whenever it first receives a client request.
+      asynchronous: false # if set to true, the router will handle the request without waiting for the coprocessor to respond
       headers: true # These boolean properties indicate which request data to include in the coprocessor request. All are optional and false by default.
       body: false
       context: false
       sdl: false
       path: false
       method: false
     response: # By including this key, the `RouterService` sends a coprocessor request whenever it's about to send response data to a client (including incremental data via @defer).
+      asynchronous: false # if set to true, the router will handle the response without waiting for the coprocessor to respond
```

Comment:
Maybe you can document it's optional in the comment

Reply:
fixed in [db39565](https://github.com/apollographql/router/pull/4902/commits/db395654b60678a5348d065b56615d86c9c0e41c)